src/cms/cms-with-local-git.js

```diff
@@ -0,0 +1,10 @@
+import CMS from 'netlify-cms-app';
+
+/**
+ * Configure CMS to allow local git editing
+ */
+CMS.init({
+  config: {
+    local_backend: true,
+  },
```

Comment:
We should be able to set it to use the github backend in localhost in here, to avoid having our emails in the identity list as that's limited on the free tier to 5 users when invite-only.

Reply:
not fully understanding? to enable github backend in localhost, you can set `NETLIFY_CMS_LOCAL_GIT=false` (see .env.example) which is the default setting

Otherwise, and what I would recommend when working locally, is to not commit directly but allow your usual git flow to review content changes.

Note that we're not limited by the number of users in Identify as we're on the pro plan (covid sponsored account), but anyways that shouldn't have anything to do with this here unless I'm misunderstanding.

Reply:
(not sure why I started this comment on this file...)

My point was you shouldn't need an extra env var for this. `NODE_ENV` should already be defined locally to `development`, if you think that's not a safe assumption we could either add it to the start script as `NODE_ENV=development gatsby develop`, or do the check backwards as `process.env.NODE_ENV !== 'production' ? cms-with-local-git.js : cms.js`. Netlify for sure sets `NODE_ENV=production` on every build (unless overriden, which we don't).

And about adding to the start script, maybe we should run the local-git on `yarn start` as well, we shouldn't need an extra terminal window for something that's just gonna sit there and show us nothing.

Reply:
And true about being on the pro plan for this, had not remembered that.